aikau/src/main/resources/alfresco/services/DialogService.js

```diff
@@ -573,7 +573,7 @@ define(["dojo/_base/declare",
             try
             {
                // Create a new pubSubScope just for this request (to allow multiple dialogs to behave independently)...
-               var pubSubScope = this.generateUuid();
+               var pubSubScope = payload.pubSubScope || this.generateUuid();
```

Comment:
This code isn't actually changing anything that has really gone before (i.e. pubSubScope configuration in payload would have had no impact). This pubSubScope generation also was added before introduction of dialogId (for cleanup/re-use). I originally added this for testing purposes but then worked around it. I can remove it, however I can see the value here for customization of not just this dialog but others.